packages/presentation/lib/src/game_bloc/bloc_screen.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/src/game_bloc/bloc.dart';
+
+abstract class BlocScreen extends StatefulWidget {
+  const BlocScreen({Key? key}) : super(key: key);
+}
+
+abstract class BlocScreenState<BS extends BlocScreen, B extends Bloc>
+    extends State<BS> {
+  BlocScreenState(Bloc this.bloc);
+  final bloc;
```

Comment:
maybe u need to put GetIt here? I saw in @alexei-kruk-idf project that he put it here like: 
  final bloc = GetIt.I.get<B>();